recipes/graphviz/yum_requirements.txt

```diff
@@ -0,0 +1 @@
+gtk2-devel
```

Comment:
Ok, so this might slow down a merge on this as yum requirements are still not supported in `conda-smithy` yet. ( https://github.com/conda-forge/conda-smithy/pull/135 )

cc @pelson  ( for the use case 😉 )